packages/babel-parser/src/plugins/typescript/index.ts

```diff
@@ -2893,8 +2894,10 @@ export default (superClass: ClassWithMixin<typeof Parser, IJSXParserMixin>) =>
     ): N.AnyExport {
       if (this.match(tt._import)) {
         // `export import A = B;`
+        const nodeImportEquals = process.env.BABEL_8_BREAKING
+          ? this.startNode<N.TsImportEqualsDeclaration>()
```

Comment:
Note that in Babel 8, `TSImportEqualsDeclaration` always starts with the `import` keyword, while the preceding `export` keyword starts the `ExportNamedDeclaration`.